docs/backup/08-01-velero-turorial.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Taking backup using Velero
+type: Tutorials
+---
+
+Follow this tutorial to install Velero and take a backup from Kyma cluster.
+
+## Prerequisites
+
+Download and install the [Velero CLI](https://github.com/heptio/velero/releases/tag/v1.3.2).
```

Comment:
Maybe just link https://github.com/vmware-tanzu/velero/releases to avoid versions?

Reply:
Done